glib.yaml

```diff
@@ -96,10 +96,9 @@ subpackages:
         - util-linux-dev # needed by gio-2.0 when building gobject-introspection
         - docbook-xml
       provides:
-        - gio-2.0
-        - glib-2.0
-        - gobject-2.0
-        - gthread-2.0
+        - gio=${{package.full-version}}
+        - gobject=${{package.full-version}}
+        - gthread=${{package.full-version}}
```

Comment:
I am questioning the need for any of these provides. Especially since we generate automatic `pc:` provides:

```
glib-dev-2.80.0-r0 provides:
cmd:gdbus-codegen=2.80.0-r0
cmd:glib-compile-resources=2.80.0-r0
cmd:glib-genmarshal=2.80.0-r0
cmd:glib-gettextize=2.80.0-r0
cmd:glib-mkenums=2.80.0-r0
cmd:gobject-query=2.80.0-r0
cmd:gresource=2.80.0-r0
cmd:gtester-report=2.80.0-r0
cmd:gtester=2.80.0-r0
gio-2.0
glib-2.0
gobject-2.0
gthread-2.0
pc:gio-2.0=2.80.0
pc:gio-unix-2.0=2.80.0
pc:girepository-2.0=2.80.0
pc:glib-2.0=2.80.0
pc:gmodule-2.0=2.80.0
pc:gmodule-export-2.0=2.80.0
pc:gmodule-no-export-2.0=2.80.0
pc:gobject-2.0=2.80.0
pc:gthread-2.0=2.80.0
```

Meaning these can just be dropped imho, given we have `pc:` ones.

Reply:
Not sure if it is needed or not. It was added long ago https://github.com/wolfi-dev/os/pull/402 by @rawlingsj.